src/evaluate/evaluator/image_classification.py

```diff
@@ -64,28 +66,31 @@ def compute(
         """
         Compute the metric for a given pipeline and dataset combination.
         Args:
-            model_or_pipeline (`str` or `Pipeline` or `Callable` or `PreTrainedModel` or `TFPreTrainedModel`,
-            defaults to `None`):
+            model_or_pipeline (`str` or `Pipeline` or `Callable` or `PreTrainedModel` or `TFPreTrainedModel`, defaults to `None`):
                 If the argument in not specified, we initialize the default pipeline for the task (in this case
                 `image-classification`. If the argument is of the type `str` or
                 is a model instance, we use it to initialize a new `Pipeline` with the given model. Otherwise we assume the
                 argument specifies a pre-initialized pipeline.
-            data (`str` or `Dataset`, defaults to `None):
+            data (`str` or `Dataset`, defaults to `None`):
                 Specifies the dataset we will run evaluation on. If it is of type `str`, we treat it as the dataset
                 name, and load it. Otherwise we assume it represents a pre-loaded dataset.
-            metric (`str` or `EvaluationModule`, defaults to `None`"
+            metric (`str` or `EvaluationModule`, defaults to `None`):
                 Specifies the metric we use in evaluator. If it is of type `str`, we treat it as the metric name, and
                 load it. Otherwise we assume it represents a pre-loaded metric.
             feature_extractor: (`str` or `FeatureExtractionMixin`, *optional*, defaults to `None`):
                 Argument can be used to overwrite a default feature extractor if `model_or_pipeline` represents a model for
                 which we build a pipeline. If `model_or_pipeline` is `None` or a pre-initialized pipeline, we ignore
                 this argument.
-            strategy: (`Literal["simple", "bootstrap"]`, defaults to "simple"):
+            strategy: (`Literal["simple", "bootstrap", "perf"]` or `List[Literal["simple", "bootstrap", "perf"]]`, defaults to "simple"):
                 specifies the evaluation strategy. Possible values are:
                 - `"simple"` - we evaluate the metric and return the scores.
                 - `"bootstrap"` - on top of computing the metric scores, we calculate the confidence interval for each
                 of the returned metric keys, using `scipy`'s `bootstrap` method
                 https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.bootstrap.html.
+                - `"perf"` - on top of computing the metric scores, we calculate the performence metrics -
```

Comment:
Maybe we could be more explicit stating that the time perfs are evaluating the metric and not the pipeline?

I think evaluation the pipeline perfs may be interesting as well but it is more "unfair" to the underlying model as the pipeline adds a bit of overhead.